doc/src/build/programming-with-objects/ch4-object-wrapping.md

```diff
@@ -68,21 +69,21 @@ public entry fun create_object(scarcity: u8, style: u8, ctx: &mut TxContext) {
 }
 ```
 
-Anyone can call `create_object` to create a new object with specified `scarcity` and `style`. The created object will be sent to the signer of the transaction. We will likely also want to be able to transfer the object to others:
+Anyone can call `create_object` to create a new object with specified `scarcity` and `style`. The created object is sent to the signer of the transaction. To enable transferring the object to others:
 
 ```rust
 public entry fun transfer_object(object: Object, recipient: address) {
     transfer::transfer(object, recipient)
 }
 ```
 
-Now let's look at how we could enable a swap/trade between your object and others' objects. A straightforward idea is this: define a function that takes two objects from two addresses and swaps their ownership. But this doesn't work in Sui! Recall from [chapter 2](ch2-using-objects.md) that only object owners can send a transaction to mutate the object. So one person cannot send a transaction that would swap their own object with someone else's object.
+You can also enable a swap/trade between your object and others' objects. For example, define a function that takes two objects from two addresses and swaps their ownership. But this doesn't work in Sui! Recall from [Using Objects](ch2-using-objects.md) that only object owners can send a transaction to mutate the object. So one person cannot send a transaction that would swap their own object with someone else's object.
 
-In the future, we will likely introduce multi-sig transactions so that two people can sign the same transaction for this type of use case. However, you may not always be able to find someone to swap with right away. A multi-sig transaction won't work in this scenario. Even if you can, you may not want to carry the burden of finding a swap target.
+Sui supports multi-signatures (multi-sig) transactions so that two people can sign the same transaction for this type of use case. But a multi-sig transaction doesn't work in this scenario. 
```

Comment:
```suggestion
Sui supports multi-signature (multi-sig) transactions so that two people can sign the same transaction for this type of use case. But a multi-sig transaction doesn't work in this scenario. 
```